src/Sentry/Internal/Constants.cs

```diff
@@ -20,6 +20,12 @@ internal static class Constants
         /// </summary>
         public const string EnvironmentEnvironmentVariable = "SENTRY_ENVIRONMENT";
 
+        /// <summary>
+        /// Default Sentry environment setting.
+        /// </summary>
+        /// <remarks>Best Sentry practice is to always try and have a value for this setting.</remarks>
+        public const string DefaultEnvironmentSetting = "Production";
```

Comment:
In ASP.NET Core this value would be `Production` but in Sentry, all SDKs will default to `production` and this value IS case sensitive. So this is a bit of a pickle

```suggestion
        public const string DefaultEnvironmentSetting = "production";
```

I'll change ASP.NET Core to use the same for the default (IsDevelopment, IsProduction) so it matches. Would you be willing to file a PR for that too?

Reply:
>  but in Sentry, all SDKs will default to production and this value IS case sensitive
So what happens if a user _manually_ sets the value to `Production` or `Development` (notice the case)? You're okay with that?

> I'll change ASP.NET Core to use the same for the default (IsDevelopment, IsProduction) so it matches

If I understand you correctly, we need to change the ASP.Net core code so that it _correctly_ sets the `.Environment` setting to be _case sensitive_ (instead of just taking the default value)?

> Would you be willing to file a PR for that too?

Yep.